### PR TITLE
Add lint for checking redundant slashes

### PR DESCRIPTION
**Pull request by LitoMore**
We have tons of URLs that have a redundant trailing slash. For example:

```diff
-https://www.1001tracklists.com
+https://www.1001tracklists.com/
```

For these domain-only URLs, we can safely remove the trailing slash.

### How do I remove those trailing slashes

The VS Code has a performance bug when I try to find all incorrect URLs with regex. So I used Sublime Text to replace texts on my local.

Here is my regex pattern:

|Find|Replace|
|:--|:--|
|`(https://([0-9a-z-]\.?)*)(/)"`|`$1"`|

### Bundle size

||Before|After|
|:--|:--|:--|
|index.js|35,434.13 KB|35,428.42 KB|

### CODE REVIEW

File: scripts/lint/ourlint.js [hunk 1]
```diff
@@ -55,6 +56,33 @@ const TESTS = {
       return `Data file is formatted incorrectly:\n\n${dataDiff}`;
     }
   },
+
+  /* Check redundant trailing slash in URL */
+  checkUrl: (data) => {
+    const hasRedundantTrailingSlash = (url) => {
+      const origin = new URL(url).origin;
+      return url.replace(origin, '') === '/';
```
Comment (mondeja) [inline scripts/lint/ourlint.js:64]: What about this example: `https://example.com//`? It passes but shouldn't. I think that the check could be "only has `/` characters and at least one".

**Review by mondeja — COMMENTED** [review]
Great. The behaviour is consistent if clients implement RFC 2616, see [this quote on Stackoverflow](https://stackoverflow.com/a/2581514/9167585).

I just have one minor suggestion.

File: scripts/lint/ourlint.js (hunk 1 above)
Comment (LitoMore) [reply scripts/lint/ourlint.js:64]: Good catch. I can change it's condition to:

```js
return /^\/+$/.test(url.repalce(origin, ''));
```

Comment (LitoMore) [reply scripts/lint/ourlint.js]: Updated.

**Review by mondeja — APPROVED** [review]
Thanks!

**Pull request merged**